modules/cloud-create-instance-steps.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/cloud-create-instance-other.adoc
+// * installing/cloud-create-instance-ocp.adoc
+
+:_content-type: PROCEDURE
+[id="cloud-create-instance-steps_{context}"]
+= Creating an instance in the console
+
+
+toc::[]
+
+[role="_abstract"]
+This topic provides the steps to create a {product-title-managed-short} instance. 
+
+.Procedure
+
+To install {product-title-managed-short}:
+
+. Log in to the {cloud-console}.
+. From the navigation menu, select *Advanced Cluster Security* -> *ACS Instances*.
+. Select *Create ACS instance* and enter information into the displayed fields or select the appropriate option from the drop-down list:
+* *Name*: Enter the name of your *ACS instance*. An *ACS instance* contains the Central resource that includes the {product-title-managed-short} management interface and services that are configured and managed by Red Hat. The Central resource communicates with your secured clusters that are managed by you. You can connect multiple secured clusters to one instance.
+* *Cloud provider*: The cloud provider where Central is located. Select *AWS*. 
+* *Cloud region*: The region for your cloud provider where Central is located. Select *AWS*.
```

Comment:
Same with these two, should it be Central custom resource?

Reply:
Leaving "Central" to be consistent with the usage in the rest of the doc.